spec/govuk_sidekiq/sidekiq_initializer_spec.rb

```diff
@@ -1,7 +0,0 @@
-require "govuk_sidekiq/sidekiq_initializer"
-
-RSpec.describe GovukSidekiq::SidekiqInitializer do
-  it "doesn't explode when called" do
-    described_class.setup_sidekiq("govuk_app_name", { url: "redis://redis" })
-  end
-end
```

Comment:
I've seen some tests like this in the past. I guess it's "it initialises without error". If an error is raised, the test should fail?

Reply:
That's probably the original intention.  I've updated the test to expect no error to be raised.